constraint-solver/src/quadratic_symbolic_expression.rs

```diff
@@ -897,6 +1013,41 @@ c = (((10 + Z) & 0xff000000) >> 24) [negative];
             effects,
             "Z: [10, 4294967050] & 0xffffff0a;
 Z: [10, 4294967050] & 0xffffffff;
+"
+        );
+    }
+
+    #[test]
+    fn solve_quadratic() {
+        let rc = RangeConstraint::from_mask(0xffu32);
+        let a = Qse::from_unknown_variable("a");
+        let b = Qse::from_known_symbol("b", rc.clone());
+        let range_constraints =
+            HashMap::from([("a", rc.clone()), ("b", rc.clone()), ("c", rc.clone())]);
```

Comment:
```suggestion
            HashMap::from([("a", rc.clone()), ("b", rc.clone())]);
```
Doesn't seem to be needed?